.vitepress/theme/style.css

```diff
@@ -149,3 +246,201 @@
 .appearance {
     display: none !important;
 }
+
+ /**
+ * Typography
+ * -------------------------------------------------------------------------- */
+/* TODO: Remove this section after Tailwind */
+
+
+h1,
```

Comment:
Shouldn't we configure some of these using tailwind config?

Reply:
Yep! that's an ideal approach. But for this update, I had to priortise getting changes live quickly. I have plans to create a single config that works accross all oyur websites and then import that config here as well. Since creating a single config for all our sites would have taken extra time, I went with this approach for now.